cmd/tools/testwick/testwicker.go

```diff
@@ -0,0 +1,246 @@
+			return w
+		}
+	}
+	return w
+}
+
+// CreateTeam creates a team in the installation
+func (w *TestWicker) CreateTeam() *TestWicker {
+	w.logger.WithField("DNS", w.installation.DNS).Info("Creating a team")
+	if w.userID == "" {
+		w.Error = fmt.Errorf("failed to create team. You need to create a user first")
+		return w
+	}
+
+	name := fmt.Sprintf("team-%d", rand.Intn(100))
+	team, response := w.mmClient.CreateTeam(&mmodel.Team{
+		Name:        name,
+		DisplayName: name,
+		Type:        mmodel.TEAM_OPEN,
+	})
+	if response.StatusCode != 201 {
+		w.Error = fmt.Errorf("failed to create a team: status code = %d, message = %s", response.StatusCode, response.Error.Message)
+		return w
+	}
+	w.teamID = team.Id
+	return w
+}
+
+// AddTeamMember add the newly created user to3 a team in the installation
+func (w *TestWicker) AddTeamMember() *TestWicker {
+	w.logger.WithField("DNS", w.installation.DNS).Info("Adding team member")
+	if w.teamID == "" {
+		w.Error = fmt.Errorf("failed to add a team member. You need to create a team first")
+		return w
+	}
+
+	_, response := w.mmClient.AddTeamMember(w.teamID, w.userID)
+	if response.StatusCode != 201 {
+		w.Error = fmt.Errorf("failed to add a team member: status code = %d, message = %s", response.StatusCode, response.Error.Message)
+		return w
+	}
+	return w
+}
+
+// CreateChannel creates a channel so we can post couple of messages
+func (w *TestWicker) CreateChannel() *TestWicker {
+	w.logger.WithField("DNS", w.installation.DNS).Info("Creating channel")
+	channel, response := w.mmClient.CreateChannel(&mmodel.Channel{
+		CreatorId: w.userID,
+		TeamId:    w.teamID,
+		Type:      mmodel.CHANNEL_OPEN,
+		Name:      fmt.Sprintf("channel-%d", rand.Intn(100)),
+	})
+	if response.StatusCode != 201 {
+		w.Error = fmt.Errorf("failed to create channel status = %d, message = %s", response.StatusCode, response.Error.Message)
+		w.logger.WithError(w.Error).Error()
+		return w
+	}
+	w.channelID = channel.Id
+	return w
+}
+
+// CreateIncomingWebhook creates an incoming webhook
+func (w *TestWicker) CreateIncomingWebhook() *TestWicker {
+	w.logger.WithField("DNS", w.installation.DNS).Info("Creating incoming webhook")
+	if w.userID == "" {
+		w.Error = fmt.Errorf("failed to post message. You need to create a user first")
+		return w
+	}
+	if w.channelID == "" {
+		w.Error = fmt.Errorf("failed to post message. You need to create channels first")
+		return w
+	}
+	_, response := w.mmClient.CreateIncomingWebhook(&mmodel.IncomingWebhook{
+		ChannelId: w.channelID,
+		UserId:    w.userID,
+	})
+	if response.StatusCode != 201 {
+		w.Error = fmt.Errorf("failed to create channel status = %d, message = %s", response.StatusCode, response.Error.Message)
+		w.logger.WithError(w.Error).Error()
+		return w
+	}
+	return w
+}
+
+// WaitForInstallationStable waits for installation to become stable
+func (w *TestWicker) WaitForInstallationStable(ctx context.Context) *TestWicker {
```

Comment:
We have some similar functions in our e2e tests https://github.com/mattermost/mattermost-cloud/blob/master/e2e/pkg/installation.go#L52
I think it might be worth it to standardize it down the line in some package, but that is outside of the scope of this PR.